mediavault/mediavault/urls.py

```diff
@@ -13,9 +13,11 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.contrib import admin
+from web.urls import urlpatterns as web_urls
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^$', include( web_urls )),
```

Comment:
And when I am doing it this way 
url(r'^', include('web.urls')
It is giving a error 
NameError: name 'web' is not defined